python-sdk/src/astro/databases/base.py

```diff
@@ -389,6 +389,18 @@ def create_schema_and_table_if_needed(
                 columns_names_capitalization=columns_names_capitalization,
             )
 
+    def fetch_all_rows(self, table: BaseTable) -> list:
```

Comment:
Couldn't it be risky to return all the rows of a specific table?
What if, by default, we had a maximum number of rows - which users could configure?

Reply:
@tataiana I added a row limit option